shell/components/Resource/Detail/Metadata/IdentifyingInformation/index.vue

```diff
@@ -39,8 +39,22 @@ const getRowValueId = (row:Row): string => `value-${ row.label }:${ row.value }`
       >
         {{ row.label }}
       </label>
+      <!-- A custom component specified as an object, responsible for it's own styling -->
       <div
-        v-if="row.valueOverride?.component && row.value"
+        v-if="typeof row.valueOverride?.component !== 'string' && row.valueOverride?.component && row.value"
+        :id="getRowValueId(row)"
+        class="full-custom-value"
+      >
+        <component
+          :is="row.valueOverride?.component"
+          v-if="row.valueOverride?.component"
+          v-bind="row.valueOverride?.props"
+          :data-testid="row.valueDataTestid"
+        />
+      </div>
+      <!-- A formatter with a component specified as a string -->
+      <div
+        v-else-if="row.valueOverride?.component && row.value"
```

Comment:
For the formatters we want to format them so that when they overflow they show elipisis.

For something like the popover the component itself needs to control overflow so the popover can be visible.

I think ideally all of the formatters would also control this but that would require significantly more work.